### DISCUSSION TITLE
Problema en Lab 01.02 Task 2

### DISCUSSION THREAD

**Discussion by mariaprojas** [rramosp/ai4eng.v1#216] [01 - INTRODUCCIÓN]
Buenos días, me sale error con este código y no sé por qué. 

![c56963ac-739d-44f0-bcf6-f282dd912097](https://user-images.githubusercontent.com/113949857/218115093-943ce2d9-9eba-449f-9ca6-55408a1104cb.jpg)

**Comment by rramosp** [comment]
estas segura de que tienes que poner average='macro' ??

On Fri, Feb 10, 2023 at 9:23 AM María Paula Rojas Ortega <
***@***.***> wrote:

> Buenos días, me sale error con este código y no sé por qué.
>
> [image: c56963ac-739d-44f0-bcf6-f282dd912097]
> <https://user-images.githubusercontent.com/113949857/218115093-943ce2d9-9eba-449f-9ca6-55408a1104cb.jpg>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/216>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZILIPS45EZMY3CUSDZTWWZFM5ANCNFSM6AAAAAAUX3NK44>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by imEag** [comment]
Hola! El problema es el parámetro average="macro". saludos
